### PR TITLE
Remove runtime section

### PR DESCRIPTION
**Pull request by trisberg**

### CODE REVIEW

File: docs/v0.4/invokers/java.md [hunk 1]
```diff
@@ -115,20 +115,21 @@ The `--handler` option is the fully qualified name of the class that provides th
...
-To deploy your function you need to select a runtime. The two options currently available are `core` and `knative` and we will select `knative`for this example:
+To deploy your function you need to select a runtime. The two options currently available are `core` and `knative` and we will select `core`for this example:
 
 ```bash
-riff knative deployer create upper --function-ref upper
+riff core deployer create upper --function-ref upper
 ```
 
-This should create the resources needed for a Knative app. You can invoke the function using the `istio-ingressgateway` service. Assuming you haven't configured DNS for your function you can use the following command to find the IP address for the `istio-ingressgateway`:
+This should create the resources needed for a deploying the function. You can invoke the function using `kubectl proxy` command to access the service that was created.
```
Comment (markfisher) [inline docs/v0.4/invokers/java.md]: s/proxy/port-forward/

Comment (trisberg) [reply docs/v0.4/invokers/java.md]: duh!

File: docs/v0.4/invokers/java.md [hunk 2]
```diff
@@ -48,14 +48,14 @@ public class UpperApplication {
 
 You can build your function either from local source or from source committed to a GitHub repository. For local build use:
 
-```bash
+```terminal
```
Comment (scothis) [inline docs/v0.4/invokers/java.md]: `terminal` isn't a support language in [highlight.js](https://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#language-names-and-aliases)

I've been using `sh`

File: docs/v0.4/invokers/java.md [hunk 3]
```diff
@@ -48,14 +48,14 @@ public class UpperApplication {
...
 riff function create upper --local-path . --handler upper
 ```
 
 When building from a GitHub repo use something like (replace the `<owner>` placeholder with your account):
 
-```bash
-riff function create upper --git-repo https://github.com/<owner>/upper-boot-gradle.git --handler upper
+```terminal
+riff function create upper --git-repo https://github.com/<owner>/upper.git --handler upper
```
Comment (scothis) [inline docs/v0.4/invokers/java.md]: Can we link to something in projectriff-samples?

File: docs/v0.4/invokers/java.md [hunk 4]
```diff
@@ -107,28 +107,30 @@ public class Hello implements Function<String, String> {
...
+```terminal
 riff function create hello --git-repo https://github.com/projectriff-samples/java-hello.git --handler functions.Hello
 ```
 
 The `--handler` option is the fully qualified name of the class that provides the function implementation.
 
 ## Deploying and invoking the function
 
-To deploy your function you need to select a runtime. The two options currently available are `core` and `knative` and we will select `knative`for this example:
+To deploy your function you need to select a runtime. The two options currently available are `core` and `knative` and we will select `core` for this example:
```
Comment (scothis) [inline docs/v0.4/invokers/java.md]: rm this whole section and link to the core runtime docs. There's nothing special about invoking a java function.

**Review by scothis — CHANGES_REQUESTED** [review]

File: docs/v0.4/invokers/java.md (hunk 2 above)
Comment (trisberg) [reply docs/v0.4/invokers/java.md]: I'd prefer something that doesn't highlight any keywords, like sh or bash highlights `function`

Comment (trisberg) [reply docs/v0.4/invokers/java.md]: maybe just leave the language out although some editors complain - https://github.com/DavidAnson/markdownlint/blob/v0.16.0/doc/Rules.md#md040

**Pull request merged**